packages/pubsub/lib/event.ts

```diff
@@ -23,6 +23,14 @@ type UserCreatedEvent = EventBase<
     }
 >;
 
-type BillingEvent = EventBase<'billing', 'billing.metric', BillingMetric[]>;
+type UsageEvent = EventBase<
+    'usage',
+    'usage.monthly_active_records' | 'usage.actions',
+    {
+        value: number;
+        accountId: number;
+        properties: Record<string, JsonValue>;
+    }
+>;
```

Comment:
instead of having specific billing events, the apps will notify about usage. Usage events will be processed for billing purpose but also to check/send email when reaching limits.